Mathlib/RingTheory/IntegralClosure/IntegralRestrict.lean

```diff
@@ -122,6 +122,22 @@ theorem galRestrict'_galLift (σ : B →ₐ[A] B₂) :
   AlgHom.ext fun x ↦ IsIntegralClosure.algebraMap_injective B₂ A L₂
     (by simp [galRestrict', Subalgebra.algebraMap_eq, galLift])
 
+variable [Algebra.IsAlgebraic K L₂] [IsFractionRing B₂ L₂] [IsFractionRing B L]
+
+/--
+A version of `galLift` for `AlgEquiv`.
+-/
+@[simps! apply symm_apply]
+noncomputable
+def galLiftEquiv (σ : B ≃ₐ[A] B₂) : L ≃ₐ[K] L₂ :=
+  AlgEquiv.ofAlgHom (galLift K L L₂ σ.toAlgHom) (galLift K L₂ L σ.symm.toAlgHom)
+  (AlgHom.coe_ringHom_injective <| IsFractionRing.ringHom_ext (A := B₂) fun x ↦ by simp)
+  (AlgHom.coe_ringHom_injective <| IsFractionRing.ringHom_ext (A := B) fun x ↦ by simp)
```

Comment:
I think you might be able to use `galLift_comp` here to get a slightly shorter proof

Reply:
Oh yeah, that's great. It makes the proof shorter and with weaker assumption. I just had to add a new simp lemma: 
```lean
theorem galLift_id : galLift K L L (.id A B) = .id K 
```